platform/runner-extension/plugin/std/sys/src/main/kotlin/io/hamal/plugin/std/sys/func/FuncGetFunction.kt

```diff
@@ -28,6 +28,14 @@ class FuncGetFunction(
                                 )
                             ),
                             "name" to StringType(func.name.value),
+                            "mcode" to MapType(
```

Comment:
what does mcode mean? I think it should just be code?

Reply:
Wanted to make a code map instead of single objects, could this make sense? Definitely would brake other tests, so called it mcode.

Reply:
Yeah, its a good call to make it a map - then lets refactor the other tests - that is okay, there will be many changes were we have to adjust tests - making code an object is definitely the way forward !